lib/edge.ig.yada/src/edge/yada/ig.clj

```diff
@@ -26,6 +26,16 @@
     id
     (assoc :id id)))
 
+(defmethod ig/init-key ::external-directories
+  [_ {:keys [path options]}]
+  (yada.resources.file-resource/new-directory-resource
```

Comment:
You need to require this namespace before using it. See `requiring-resolve` below.